deploy/src/main/java/com/tc/server/util/TcHashSessionIdManager.java

```diff
@@ -86,98 +98,62 @@ public void invalidateAll(String id) {
     getDelegate().invalidateAll(id);
   }
 
+  @Override
+  public String newSessionId(Request request, String s, long l) {
+    return "";
+  }
+
   @Override
   public boolean isFailed() {
-    if (delegate == null) { return false; }
     return getDelegate().isFailed();
   }
 
   @Override
   public boolean addEventListener(EventListener eventListener) {
-    if (delegate == null) { return false; }
     return getDelegate().addEventListener(eventListener);
   }
 
   @Override
   public boolean removeEventListener(EventListener eventListener) {
-    if (delegate == null) { return false; }
     return getDelegate().removeEventListener(eventListener);
   }
 
   @Override
   public final void start() throws Exception {
-    if (delegate != null) { return; }
-
-    // Initialize delegate lazily to prevent the SecureRandom
-    // it contains from blocking the jetty initialization.
-    Thread thread = new Thread() {
-      @Override
-      public void run() {
-        getDelegate();
-      }
-    };
-    thread.setName("TcHashSessionIdManager initializer");
-    thread.setDaemon(true);
-    thread.start();
+    getDelegate().start();
```

Comment:
May need to reinstate this delayed start -- `TcHashSessionIdManager` does use a `SecureRandom` (which can take a long time to initialize) but `TcHashSessionIdManager` currently appears unused.